src/Lattice/model_lib/model_heisenberg.jl

```diff
@@ -5,8 +5,8 @@
         ) :: Nothing
 
 Init Heisenberg model on a given lattice by overwriting the respective bonds.
-Here, J[n] is the coupling to the (n - 1)-th nearest neighbor (Euclidean norm, n = 1 corresponds to on-site interaction).
-If there are m symmetry inequivalent (n - 1)-th nearest neighbors, these are
+Here, J[n] is the coupling to the (n)-th nearest neighbor (Euclidean norm).
```

Comment:
Can we remove the bracket around the n?

Reply:
Actually like it with brackets more, but no hard feelings about leaving them out
```suggestion
Here, J[n] is the coupling to the n-th nearest neighbor (Euclidean norm).
```